src/platform/Zephyr/ConfigurationManagerImpl.cpp

```diff
@@ -188,7 +186,16 @@ void ConfigurationManagerImpl::DoFactoryReset(intptr_t arg)
 CHIP_ERROR ConfigurationManagerImpl::GetPrimaryWiFiMACAddress(uint8_t * buf)
 {
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFI
-    return WiFiManager::Instance().GetMACAddress(buf);
+    auto getInterface = [](Inet::InterfaceId ifaceId = Inet::InterfaceId::Null()) {
```

Comment:
Maybe we need some sort of `WiFiUtil.h/cpp` in `platform/Zephyr` that would provide this function? Note that although we currently use `net_if_get_default`, we may want to apply a more sophisticated logic at some point to make sure that if both WiFi and OpenThread interfaces exist, the WiFi one is selected. Thus it would be good to have it aligned.